Release Notes/2018/2018-04-10-cloud-platform-release-notes.md

```diff
@@ -16,9 +16,9 @@ Application Lifecycle Management now offers a Workload Map view into the instanc
 
 ### Enhancements (1)
 
-#### [DCC Foundation](//www.ctl.io/dcc-foundation/)
+#### [CenturyLink Private Cloud on VMware Cloud Foundation](/centurylink-private-cloud-on-vmware-cloud-foundation)
```

Comment:
change `/centurylink-private-cloud-on-vmware-cloud-foundation`
to
`//www.ctl.io/centurylink-private-cloud-on-vmware-cloud-foundation/`